code/online_challenges/src/codechef/GIFTSRC/GIFTSRC.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+using namespace std;
+
+int main() {
```

Comment:
Reformat this file with clang format and 4 spaces. loop variables should be declared in the loop itself. All other variables should be declared closest to where they are used.